.craftplugin

```diff
@@ -1 +1 @@
-{"pluginName":"Website Toolbox Community","pluginDescription":"Your CraftCMS forum doesn’t need to be basic. Website Toolbox is a cloud-based community plugin that allows you to easily add a powerful, intuitive, and maintenance-free community to CraftCMS without slowing it down. No database, servers, or coding required","pluginVersion":"1.4.0","pluginAuthorName":"Websitetoolbox","pluginVendorName":"websitetoolbox","pluginAuthorUrl":"http://websitetoolbox.com","pluginAuthorGithub":"","codeComments":"","pluginComponents":["services","settings"],"consolecommandName":"","controllerName":"","cpsectionName":"","elementName":"","fieldName":"","modelName":"","purchasableName":"","recordName":"","serviceName":"","taskName":"","utilityName":"","widgetName":"","apiVersion":"api_version_3_0"}
\ No newline at end of file
+{"pluginName":"Website Toolbox Community","pluginDescription":"Your CraftCMS community doesn’t need to be basic. Website Toolbox is a cloud-based community plugin that allows you to easily add a powerful, intuitive, and maintenance-free community to CraftCMS without slowing it down. No database, servers, or coding required","pluginVersion":"1.4.0","pluginAuthorName":"Websitetoolbox","pluginVendorName":"websitetoolbox","pluginAuthorUrl":"http://websitetoolbox.com","pluginAuthorGithub":"","codeComments":"","pluginComponents":["services","settings"],"consolecommandName":"","controllerName":"","cpsectionName":"","elementName":"","fieldName":"","modelName":"","purchasableName":"","recordName":"","serviceName":"","taskName":"","utilityName":"","widgetName":"","apiVersion":"api_version_3_0"}
```

Comment:
You didn't do what this previous comment asked you to do:
"Change the whole description to this:
The Easiest CraftCMS Community Plugin. Effortlessly Build a Beautiful Community Forum with Zero Coding, Seamless Embedding, Instant Setup, and Exceptional Support."